### PR TITLE
Add compat data for HTMLCanvasElement

### PR DESCRIPTION
**Pull request by maboa**

### CODE REVIEW

File: api/HTMLCanvasElement.json [hunk 1]
```diff
@@ -0,0 +1,757 @@
...
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": false,
+            "deprecated": false
+          }
+        },
+        "webgl context": {
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/HTMLCanvasElement.json]: 👍

[the remark this answers is not in the document]
Comment (maboa) [reply api/HTMLCanvasElement.json]: 👍

File: api/HTMLCanvasElement.json [hunk 2]
```diff
@@ -0,0 +1,757 @@
...
+          "__compat": {
+            "support": {
+              "webview_android": {
+                "version_added": null
+              },
+              "chrome": {
+                "version_added": "9",
+                "notes": "Chrome 9 and Firefox 3.6 initially implemented this as <code>experimental-webgl</code>. Since Chrome 33 and Firfox 24 it is implemented as the standard <code>webgl</code>."
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/HTMLCanvasElement.json]: 👍

File: api/HTMLCanvasElement.json [hunk 3]
```diff
@@ -0,0 +1,757 @@
...
+              },
+              "chrome_android": {
+                "version_added": null
+              },
+              "edge": {
+                "version_added": true,
+                "notes": "Edge, Internet Explorer 11, WebKit 5.1 and Firefox Mobile implemented this as <code>experimental-webgl</code>."
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/HTMLCanvasElement.json]: :+1:

File: api/HTMLCanvasElement.json [hunk 4]
```diff
@@ -0,0 +1,757 @@
...
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": [
+                {
+                  "version_added": "3.6",
+                  "notes": "Chrome 9 and Firefox 3.6 initially implemented this as <code>experimental-webgl</code>. Since Chrome 33 and Firfox 24 it is implemented as the standard <code>webgl</code>."
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/HTMLCanvasElement.json]: 👍

File: api/HTMLCanvasElement.json [hunk 5]
```diff
@@ -0,0 +1,757 @@
...
+                },
+                {
+                  "version_added": "24"
+                }
+              ],
+              "firefox_android": {
+                "version_added": true,
+                "notes": "Edge, Internet Explorer 11, WebKit 5.1 and Firefox Mobile implemented this as <code>experimental-webgl</code>."
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/HTMLCanvasElement.json]: 👍

File: api/HTMLCanvasElement.json [hunk 6]
```diff
@@ -0,0 +1,757 @@
...
+              },
+              "ie": {
+                "version_added": "11",
+                "notes": "Edge, Internet Explorer 11, WebKit 5.1 and Firefox Mobile implemented this as <code>experimental-webgl</code>."
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/HTMLCanvasElement.json]: :+1:

File: api/HTMLCanvasElement.json [hunk 7]
```diff
@@ -0,0 +1,757 @@
...
+              },
+              "opera": {
+                "version_added": "9",
+                "notes": "Opera 9 implemented this as <code>experimental-webgl</code>, behind a user preference, in version 15.0 the user preference got removed."
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/HTMLCanvasElement.json]: :+1:

File: api/HTMLCanvasElement.json [hunk 8]
```diff
@@ -0,0 +1,757 @@
...
+              },
+              "opera_android": {
+                "version_added": null
+              },
+              "safari": {
+                "version_added": "5.1",
+                "notes": "Edge, Internet Explorer 11, WebKit 5.1 and Firefox Mobile implemented this as <code>experimental-webgl</code>."
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/HTMLCanvasElement.json]: :+1: Made it an `alternative_name` like the others.

File: api/HTMLCanvasElement.json [hunk 9]
```diff
@@ -0,0 +1,757 @@
...
+              },
+              "safari_ios": {
+                "version_added": null
+              }
+            }
+          }
+        },
+        "webgl2 context": {
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/HTMLCanvasElement.json]: 👍

File: api/HTMLCanvasElement.json [hunk 10]
```diff
@@ -0,0 +1,757 @@
...
+              "edge": {
+                "version_added": false,
+                "notes": "WebGL 2 is under consideration in Edge, with a roadmap priority of medium. See the <a href='https://developer.microsoft.com/en-us/microsoft-edge/platform/status/webgl20/'></a>Edge Platform status for latest information."
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": "25",
+                "notes": "Firefox 25 implemented this as <code>experimental-webgl2</code> behind the user preference webgl.enable-prototype-webgl2. Starting with Firefox 25, the string <code>webgl2</code> is used behind the same preference and <code>experimental-webgl2</code> is no longer accepted"
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/HTMLCanvasElement.json]: Trying to keep consistent by using `alternative_name` - will push and let you comment though.

File: api/HTMLCanvasElement.json [hunk 11]
```diff
@@ -0,0 +1,757 @@
...
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              }
+            }
+          }
+        },
+        "2d alpha context attribute": {
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/HTMLCanvasElement.json]: 👍

File: api/HTMLCanvasElement.json [hunk 12]
```diff
@@ -0,0 +1,757 @@
...
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              }
+            }
+          }
+        },
+        "failIfMajorPerformanceCaveat attribute": {
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/HTMLCanvasElement.json]: :+1:

File: api/HTMLCanvasElement.json [hunk 13]
```diff
@@ -0,0 +1,757 @@
...
+              "notes": "See <a href='https://bugs.webkit.org/show_bug.cgi?id=71270'>WebKit bug 71270</a>."
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/HTMLCanvasElement.json]: 👍

File: api/HTMLCanvasElement.json [hunk 14]
```diff
@@ -0,0 +1,757 @@
...
+          }
+        },
+        "Image quality parameter": {
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/HTMLCanvasElement.json]: 👍

File: api/HTMLCanvasElement.json [hunk 15]
```diff
@@ -0,0 +1,835 @@
...
+              "version_added": "44",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "gfx.offscreencanvas.enabled"
+                }
+              ]
+            },
+            "firefox_android": {
+              "version_added": "4",
```
Comment (Elchi3) [inline api/HTMLCanvasElement.json]: 44

File: api/HTMLCanvasElement.json [hunk 16]
```diff
@@ -0,0 +1,835 @@
...
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "Image quality parameter": {
```
Comment (Elchi3) [inline api/HTMLCanvasElement.json]: No spaces in the identifier; please add a description.

**Comment by maboa** [comment]
done and done :)

**Pull request merged**